IRremoteESP8266.cpp

```diff
@@ -750,25 +749,32 @@ static void ICACHE_RAM_ATTR gpio_intr() {
   os_timer_disarm(&timer);
   GPIO_REG_WRITE(GPIO_STATUS_W1TC_ADDRESS, gpio_status);
 
-  if (irparams.rawlen >= RAWBUF) {
+  // Grab a local copy of rawlen to reduce instructions used in IRAM.
+  // This is an ugly premature optimisation code-wise, but we do everything we
+  // can to save IRAM.
+  // It seems referencing the value via the structure uses more instructions.
+  // Less instructions means faster and less IRAM used.
+  // N.B. It saves about 13 bytes of IRAM.
+  uint16_t rawlen = irparams.rawlen;
```

Comment:
But we do want the interrupt to be in iram so it can be fast as possible?

How did you do the check how many instructions were saved so I could do the same? Is it from the Arduino ide?